storage/client/pom.xml

```diff
@@ -98,7 +112,35 @@
   </dependencies>
 
   <build>
+
+    <sourceDirectory>src/main/java</sourceDirectory>
+    <testSourceDirectory>src/test/java</testSourceDirectory>
+
     <plugins>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <compilerId>groovy-eclipse-compiler</compilerId>
```

Comment:
I did not see these in client pom. We have ran the tests under current pom, I am wondering the groovy plugin needed to run the tests.

Reply:
Yeah without this mvn test will not trigger any of the tests.